crates/sui-snapshot/src/reader.rs

```diff
@@ -301,18 +325,18 @@ impl StateSnapshotReaderV1 {
         );
         let obj_progress_bar_clone = obj_progress_bar.clone();
         let downloaded_bytes = AtomicUsize::new(0);
-        let file_counter = Arc::new(AtomicUsize::new(0));
         let instant = Instant::now();
 
-        let ret = Abortable::new(
+        Abortable::new(
```

Comment:
nit: maybe this whole block of code could be made a function as well

Reply:
Done